windows/src/developer/kmcmpdll/Compiler.cpp

```diff
@@ -3497,3 +3497,9 @@ HANDLE UTF16TempFromUTF8(HANDLE hInfile, BOOL hasPreamble)
 	SetFilePointer(hOutfile, 2, NULL, FILE_BEGIN);
 	return hOutfile;
 }
+
+extern "C" void __declspec(dllexport) WINAPI Keyman_Diagnostic(int mode) {
+  if (mode == 0) {
+    RaiseException(0x0EA0BEEF, EXCEPTION_NONCONTINUABLE, 0, NULL);
```

Comment:
why no `0xDEADBEEF`? 🐮

Reply:
Because Windows does not allow us to raise an exception with that code (bit 28 is reserved and Windows clears it if we use it). We could find another code ... but importance seems low.